packages/forma-36-react-components/package.json

```diff
@@ -13,8 +13,7 @@
   "browserslist": "extends @contentful/browserslist-config",
   "scripts": {
     "build:clean": "rm -rf dist",
-    "build": "yarn build:clean && microbundle src/index.ts && yarn size",
-    "size": "size-limit",
+    "build": "yarn build:clean && microbundle src/index.ts",
```

Comment:
Q: why are we using `microbundle` (Rollup) only for this package and not parcel like others?

Reply:
I've no idea, I tried to track it down and it comes from the original v4 pull request. I wanted to look into it in a separate PR 🤷